tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -242,8 +252,9 @@ public function theNotificationsShouldBeEmpty(): void {
 	 * @throws Exception
 	 */
 	public function userShouldNotHaveAnyNotification($user): void {
-		$this->userListAllNotifications($user);
-		$this->theNotificationsShouldBeEmpty();
+		$response = $this->listAllNotifications($user);
+		$notifications = json_decode($response->getBody()->getContents())->ocs->data;
```

Comment:
do we have method to get ocs data?
```suggestion
		$notifications = getOCSData();
```